docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -0,0 +1,7 @@
+= Configuration Properties
```

Comment:
looks confusing, like it relates to a configuraiton.properties when it relates to remote tests execution